openshift-templates/project-acl/template.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Can you merge this template with https://github.com/rht-labs/labs-ci-cd/blob/master/openshift-templates/rolebinding-group/template.yaml